src/router.ts

```diff
@@ -123,6 +124,17 @@ export default class Router {
       }
     });
 
+    router.get('/addresses', async (req, res) => {
```

Comment:
I'd probably prefer this endpoint to have a more specific route name like `'/marketmaker/addresses'`.